libmpv/stream_cb.h

```diff
@@ -147,6 +147,19 @@ typedef int64_t (*mpv_stream_cb_size_fn)(void *cookie);
  */
 typedef void (*mpv_stream_cb_close_fn)(void *cookie);
 
+/**
+ * Cancel callback used to implement a custom stream.
+ *
+ * This callback will be called from a separate thread from the demux
+ * thread.
```

Comment:
This should say what it actually does (e.g. interrupting current and future read or seek operations).
It should also say that this callback must not block.

Also, each API addition should bump MPV_CLIENT_API_VERSION.